src/components/chat_lists/ChatList.module.css

```diff
@@ -0,0 +1,12 @@
+.chats {
+    background-color: white;
+    border-top: 1px solid darkgrey;
```

Comment:
move all color to a separate file, and use variables instead of hardcoded values